src/Database/DBM.php

```diff
@@ -114,19 +114,13 @@ public static function esc_array(&$arr, $add_quotation = false)
 	/**
 	 * Checks Converts any date string into a SQL compatible date string
 	 *
+	 * @deprecated since version 3.6
 	 * @param string $date a date string in any format
 	 *
 	 * @return string SQL style date string
 	 */
 	public static function date($date = 'now')
 	{
-		$timestamp = strtotime($date);
-
-		// Don't allow lower date strings as '0001-01-01 00:00:00'
-		if ($timestamp < -62135596800) {
-			$timestamp = -62135596800;
-		}
-
-		return date(DateTimeFormat::MYSQL, (int)$timestamp);
+		return DateTimeFormat::utc($date);
```

Comment:
The special thing about the old function is to check for the minimal date that is allowed. I think that this has to be added somewhere.

Reply:
I believe it is already right here: https://github.com/friendica/friendica/blob/develop/src/Util/DateTimeFormat.php#L111

Reply:
This looks really, really ugly to me, since you can use keywords for this function that might interfere. Have you checked if this works?

Reply:
We using are using keywords explicitly, so there are no reasons to believe we would intentionally go before 0001.